.changesets/docs_gigi_dxm_227_add_railway_and_render_deployment_guides.md

```diff
@@ -0,0 +1,5 @@
+### add render and railway deployment pages ([PR #8242](https://github.com/apollographql/router/pull/8242))
+
+[dxm-227](https://apollographql.atlassian.net/browse/DXM-227)
```

Comment:
I don't think we want to add any Apollo-only ticket links to public changesets. Is this something we can still remove? 

I believe we also don't typically add docs-only changes to a changeset... @abernix would you be able to confirm here? 🙏🏽